core/imagedownloader.cpp

```diff
@@ -44,6 +44,7 @@ bool ImageDownloader::loadFromUrl(const QUrl &url)
 		connect(&manager, &QNetworkAccessManager::finished, this,
 			[this,&success] (QNetworkReply *reply) { saveImage(reply, success); });
 		connect(&manager, &QNetworkAccessManager::finished, &loop, &QEventLoop::quit);
+		qDebug() << "Downloading image from " << url;
```

Comment:
[1] sorry, my comment was about the extra space being redundant for all of these `qDebug()` calls.
```qDebug() << "Downloading image from" << url;```